Makefile

```diff
@@ -4,12 +4,20 @@ all:
 	$(MAKE) build
 
 .PHONY: get-deps
-get-deps:
+get-deps: get-deps-pip get-deps-proto get-deps-webui get-deps-go
```

Comment:
Have separate `get-deps-webui-elm` and `get-deps-webui-react` dependencies (use `subst` to turn `-` into `/`, like the other places).

Reply:
I'll look up how to use substitution. we've had issues with the node caches colliding in the past so I'd probably lean on the safer side and not separate those.